app/src/main/java/com/twilio/video/app/sdk/LocalParticipantListener.kt

```diff
@@ -23,18 +21,7 @@ class LocalParticipantListener(private val roomManager: RoomManager) : LocalPart
         roomManager.sendRoomEvent(NetworkQualityLevelChange(localParticipant.sid, networkQualityLevel))
     }
 
-    override fun onVideoTrackPublished(localParticipant: LocalParticipant, localVideoTrackPublication: LocalVideoTrackPublication) {
```

Comment:
So this is no longer needed since any changes to the LocalVideoTrack are communicated via [this line](https://github.com/twilio/twilio-video-app-android/blob/c5fbd93226dd219f9c75d347fa4f2af4c523a131/app/src/main/java/com/twilio/video/app/sdk/LocalParticipantManager.kt#L41) in the `LocalParticipantManager`.